### ISSUE TITLE
[PATCH] Fix typos (spelling errors) in NetWare/*.

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#11011]
Migrated from [rt.perl.org#81898](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=81898) (status was 'resolved')

Searchable as RT81898$

**Comment by p5pRT** [comment]
### From @pjacklam



---
 NetWare/NWUtil\.c       |    2 \+\-
 NetWare/Nwmain\.c       |    2 \+\-
 NetWare/config\_H\.wc    |   10 \+\+\+\+\+\-\-\-\-\-
 NetWare/nwplglob\.c     |    2 \+\-
 NetWare/nwvmem\.h       |    2 \+\-
 NetWare/t/NWScripts\.pl |    6 \+\+\+\-\-\-
 6 files changed\, 12 insertions\(\+\)\, 12 deletions\(\-\)

<details><summary>Inline Patch</summary>

```diff
diff --git a/NetWare/NWUtil.c b/NetWare/NWUtil.c
index f23774a..8db93c6 100644
--- a/NetWare/NWUtil.c
+++ b/NetWare/NWUtil.c
@@ -11,7 +11,7 @@
  * FILENAME		:	NWUtil.c
  * DESCRIPTION	:	Utility functions for NetWare implementation of Perl.
  * Author		:	HYAK
- * Date			:	Januray 2001.
+ * Date			:	January 2001.
  *
  */
 
diff --git a/NetWare/Nwmain.c b/NetWare/Nwmain.c
index b49fc80..a64534e 100644
--- a/NetWare/Nwmain.c
+++ b/NetWare/Nwmain.c
@@ -169,7 +169,7 @@ void main(int argc, char *argv[])
 
 	// Create the file NUL if not present. This is done only once per NLM load.
 	// This is required for -e.
-	// Earlier verions were creating temporary files (in perl.c file) for -e.
+	// Earlier versions were creating temporary files (in perl.c file) for -e.
 	// Now, the technique of creating temporary files are removed since they were
 	// fragile or insecure or slow. It now uses the memory by setting
 	// the BIT_BUCKET to "nul" on Win32, which is equivalent to /dev/nul of Unix.
diff --git a/NetWare/config_H.wc b/NetWare/config_H.wc
index bd9b62a..5fc6a84 100644
--- a/NetWare/config_H.wc
+++ b/NetWare/config_H.wc
@@ -973,7 +973,7 @@
 
 /* HAS_QUAD:
  *	This symbol, if defined, tells that there's a 64-bit integer type,
- *	Quad_t, and its unsigned counterpar, Uquad_t. QUADKIND will be one
+ *	Quad_t, and its unsigned counterpart, Uquad_t. QUADKIND will be one
  *	of QUAD_IS_INT, QUAD_IS_LONG, QUAD_IS_LONG_LONG, or QUAD_IS_INT64_T.
  */
 /*#define HAS_QUAD	/**/
@@ -1375,7 +1375,7 @@
 
 /* HAS_GETESPWNAM:
  *	This symbol, if defined, indicates that the getespwnam system call is
- *	available to retrieve enchanced (shadow) password entries by name.
+ *	available to retrieve enhanced (shadow) password entries by name.
  */
 /*#define HAS_GETESPWNAM		/**/
 
@@ -1725,7 +1725,7 @@
 
 /* HAS_MKSTEMPS:
  *	This symbol, if defined, indicates that the mkstemps routine is
- *	available to excluslvely create and open a uniquely named
+ *	available to exclusively create and open a uniquely named
  *	(with a suffix) temporary file.
  */
 /*#define HAS_MKSTEMPS		/**/
@@ -3101,7 +3101,7 @@
  *	unsigned long, int, etc.  It may be necessary to include
  *	<sys/types.h> to get any typedef'ed information.
  */
-#define Size_t size_t	 /* length paramater for string functions */
+#define Size_t size_t	 /* length parameter for string functions */
 
 /* Sock_size_t:
  *	This symbol holds the type used for the size argument of
@@ -3305,7 +3305,7 @@
  *	    2 = supports arrays of pointers to functions returning void
  *	    4 = supports comparisons between pointers to void functions and
  *		    addresses of void functions
- *	    8 = suports declaration of generic void pointers
+ *	    8 = supports declaration of generic void pointers
  *
  *	The package designer should define VOIDUSED to indicate the requirements
  *	of the package.  This can be done either by #defining VOIDUSED before
diff --git a/NetWare/nwplglob.c b/NetWare/nwplglob.c
index c7a2104..51a3e5e 100644
--- a/NetWare/nwplglob.c
+++ b/NetWare/nwplglob.c
@@ -9,7 +9,7 @@
 
 /*
  * FILENAME		:	nwplglob.c
- * DESCRIPTION	:	Perl globbing support for NetWare. Other platforms have usually lauched
+ * DESCRIPTION	:	Perl globbing support for NetWare. Other platforms have usually launched
  *                  a separate executable for this in order to take advantage of their
  *                  shell's capability for generating a list of files from a given
  *                  wildcard file spec. On NetWare, we don't have that luxury.
diff --git a/NetWare/nwvmem.h b/NetWare/nwvmem.h
index 5999d88..da41afd 100644
--- a/NetWare/nwvmem.h
+++ b/NetWare/nwvmem.h
@@ -13,7 +13,7 @@
  *                  Watcom's hash table is used to store memory pointers.
  *                  All malloc's, realloc's, free's go through this.
  * Author		:	HYAK, SGP
- * Date			:	Januray 2001.
+ * Date			:	January 2001.
  *
  */
 
diff --git a/NetWare/t/NWScripts.pl b/NetWare/t/NWScripts.pl
index a594dac..c16a4a1 100644
--- a/NetWare/t/NWScripts.pl
+++ b/NetWare/t/NWScripts.pl
@@ -24,7 +24,7 @@ foreach $DirItem(@Dirs)
 	if(-d $DirItem1)
 	{	# If an item is a folder, then open it further.
 
-		# Intemediary automated script like base.pl, lib.pl, cmd.pl etc.
+		# Intermediary automated script like base.pl, lib.pl, cmd.pl etc.
 		$IntAutoScript = "t/".$DirItem.".pl";
 
 		# Open once in write mode since later files are opened in append mode,
@@ -61,7 +61,7 @@ foreach $FileName(@DirNames)
 			{
 				$DirItem = $DirItem1;
 
-				# Intemediary automated script like base.pl, lib.pl, cmd.pl etc.
+				# Intermediary automated script like base.pl, lib.pl, cmd.pl etc.
 				$IntAutoScript = "t/".$DirItem.".pl";
 			}
 		}
@@ -160,7 +160,7 @@ foreach $DirItem(@Dirs)
 		{
 			# The folder not empty.
 
-			# Intemediary automated script like base.pl, lib.pl, cmd.pl etc.
+			# Intermediary automated script like base.pl, lib.pl, cmd.pl etc.
 			$IntAutoScript = "t/".$DirItem.".pl";
 
 			# Write into the intermediary auto script.
-- 
1.7.3.3
```
</details>

**Comment by p5pRT** [comment]
### From @abigail

On Thu\, Jan 06\, 2011 at 11&#8203;:13&#8203;:17PM \-0800\, Peter J\. Acklam via RT wrote&#8203;:
> \# New Ticket Created by  \(Peter J\. Acklam\) 
> \# Please include the string&#8203;:  \[perl \#81898\]
> \# in the subject line of all future correspondence about this issue\. 
> \# \<URL&#8203;: http&#8203;://rt\.perl\.org/rt3/Ticket/Display\.html?id=81898 >
> 

Thanks\, applied as a00faafa3359b919af63c5dc4f58b202c327189d

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
@iabyn - Status changed from 'open' to 'resolved'
